aws/client/logger.go

```diff
@@ -90,13 +130,15 @@ func logResponse(r *request.Request) {
 			return
 		}
 
-		b, err := ioutil.ReadAll(lw.buf)
-		if err != nil {
-			lw.Logger.Log(fmt.Sprintf(logRespErrMsg, req.ClientInfo.ServiceName, req.Operation.Name, err))
-			return
-		}
 		lw.Logger.Log(fmt.Sprintf(logRespMsg, req.ClientInfo.ServiceName, req.Operation.Name, string(body)))
```

Comment:
`lw` wouldn't be initialized here when it should of been if `LogDebugWithHTTPBody` isn't set.